functional-samples/sample.webgpu/README.md

```diff
@@ -0,0 +1,17 @@
+# WebGPU extension sample
+
+This sample demonstrates how to use the [WebGPU API](https://webgpu.dev/) to generate a red triangle using a service worker.
+
+> [!WARNING]  
+> Service worker support in WebGPU is available in Chrome 123 with the "Experimental Web Platform Features" flag.
```

Comment:
When is this expected to be available without a flag? (I'm not asking for that information to be added.)

Reply:
Hopefully Chrome 124 once I sent the intent to ship next week.